test/utils/throwingInstance.ts

```diff
@@ -0,0 +1,23 @@
+import { instance } from "ts-mockito";
+
+/**
+ * Creates an instance of a mocked object. Calls to methods or properties that have not been stubbed will throw errors
+ */
+export default function throwingInstance<T>(target: T) {
```

Comment:
I forgot to mention this last time, but it's good to call the T variable something a bit more meaningful. Maybe TMock would be good in this case.